packages/contracts-bedrock/src/cannon/PreimageOracle.sol

```diff
@@ -51,8 +51,8 @@ contract PreimageOracle is ISemver {
     uint256 public constant PRECOMPILE_CALL_RESERVED_GAS = 100_000;
 
     /// @notice The semantic version of the Preimage Oracle contract.
-    /// @custom:semver 1.1.3-beta.6
-    string public constant version = "1.1.3-beta.6";
+    /// @custom:semver 1.1.3-beta.7
+    string public constant version = "1.1.3-beta.7";
```

Comment:
accidental semver bump?

Reply:
Nope, lib-keccak was updated

Reply:
Yeah it's actually on the to-do @smartcontracts

Reply:
CI will at least force you to semver bump when dependencies are changed

Reply:
We are planning to simplify things and remove contract-specific semvers as part of the OPCM Upgrades work in https://github.com/ethereum-optimism/design-docs/pull/159. That design review is today (at 1030am PT)